cli/js2/000_bootstrap_namespace.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+// The only purpose of this file is to set up "window.__bootstrap" namespace,
+// that is used by scripts in this directory to reference exports between
+// the files.
+
+// This namespace is removed during runtime bootstrapping process.
+
+((window) => {
+  window.__bootstrap = {};
+})(this);
```

Comment:
Two things:
1. I don't think the anonymous closure is useful here.
2. Why not use `window.bootstrap` instead of `window.__bootstrap`?

Reply:
1. Removed closure.
2. It's not, please read comment.